implants/lib/eldritch/src/pivot/ssh_copy_impl.rs

```diff
@@ -63,14 +63,11 @@ pub fn ssh_copy(
     )) {
         Ok(local_res) => local_res,
         Err(local_err) => {
-            return Err(anyhow::anyhow!(
-                "Failed to run handle_ssh_exec: {}",
-                local_err.to_string()
-            ))
+            return Ok(format!("Failed to run handle_ssh_copy: {}", local_err));
```

Comment:
What's your opinion on maybe using `ReportErrorMessage` to report the Error and then return a bool for success? It feels sort of janky to be melding the return value as a string with how people will react on failure. I would note we may also need to make a note in the docs that unlike all other functions this one will not end the tome on failure.

Reply:
This is how we do command / shell exec.

Also starlark gets Madge when you try to do both eval and heap so unsure how to do a dict return type and emit errors.

Reply:
Fair enough for now. May revisit at some point